assets/dev/js/editor/document/hooks/data/document/elements/delete/create-column-section-empty.js

```diff
@@ -0,0 +1,45 @@
+import After from 'elementor-api/modules/hooks/data/after';
```

Comment:
Why create a new hook instead of fixing the old one?

Reply:
Its not deleting, its renaming since the old name does not represent the hook purpose.

Reply:
For separate logic and more readable code.